demo/demo.md

```diff
@@ -1,233 +1,312 @@
-# Alert
-
+<!--
+The demo.md file is a compiled document. No edits should be made directly to this file.
+README.md is created by running `npm run build:docs`.
+This file is generated based on a template fetched from `./docs/partials/demo.md`
+-->
+
+# accordion
```

Comment:
Change references from "accordion" to "alert"

Reply:
oops, my bad.